entries/i/ing.nl.json

```diff
@@ -0,0 +1,23 @@
+{
+  "ING": {
```

Comment:
This should include some country idenfication, e.g. `ING (NL)`, `ING (Netherlands)` or `ING (Nederland)`.